src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/HearingScheduledClaimantScenarioTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.reform.civil.controllers.dashboard.scenarios.claimant;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.http.HttpStatus;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import uk.gov.hmcts.reform.civil.controllers.BaseIntegrationTest;
+import uk.gov.hmcts.reform.dashboard.data.ScenarioRequestParams;
+import uk.gov.hmcts.reform.dashboard.data.TaskStatus;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.DashboardScenarios.SCENARIO_AAA6_CP_HEARING_SCHEDULED_CLAIMANT;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@Testcontainers
+public class HearingScheduledClaimantScenarioTest extends BaseIntegrationTest {
+
+    @Test
+    void should_create_hearing_scheduled_scenario() throws Exception {
+        UUID caseId = UUID.randomUUID();
+        doPost(BEARER_TOKEN,
```

Comment:
Yes, but from the handler it is calling the locations API (this is necessary to get the correct name of the hearing court). If the integration test is done by calling the handler, locationsService returns an error because it does not recognize the auth token, so the test cannot be completed. That's why I have done it by POST